src/xo-app/host/tab-patches.js

```diff
@@ -111,6 +115,34 @@ const INSTALLED_PATCH_COLUMNS_2 = [
   needsRestart: createDoesHostNeedRestart((_, props) => props.host)
 }))
 export default class HostPatches extends Component {
+  _installPatchWarning = ({patch, installPatch}) => confirm(
+    {
+      title: _('installPatchWarningTitle'),
+      body: <div>
+        <p>{_('installPatchWarningContent')}</p>
+        <Link to={'/pools/' + this.props.host.$pool + '/patches'}>
+          <Button
+            btnStyle='success btn-block'
+            onClick={close}>Go to pool level patching
```

Comment:
All the messages should be provided by `intl` so they can be translated.

And this should be indented like this:
```js
<Button
  ...
>
  {_('myMessage')}
</Button>
```